<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="products_tab">
       <attribute name="title">
        <string>Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QTableView" name="secondSet_tbl">
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="sortSets_btn">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>sort all products tables</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="firstSet_tbl">
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="mergedSet_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>merged</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QTableView" name="mergedSet_tbl"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="secondSet_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>2nd set</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="firstSet_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>1st set</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="mergeSets_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>merge and sort sets</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bags_tab">
       <attribute name="title">
        <string>Bags</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="firstBag_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>1st bag</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="secondBag_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>2nd bag</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="mergedBags_lbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>merged</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="firstBag_tbl">
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableView" name="secondBag_tbl">
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QTableView" name="mergedBags_tbl"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="sortBags_btn">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>sort all bags</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="mergeBags_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>merge and sort bags</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="enterWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QSplitter" name="enterPart">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>156</y>
         <width>151</width>
         <height>321</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="enterProd1_lbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>products, 1st set:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prod1_edit">
        <property name="text">
         <string>products1.txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="enterProd2_lbl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>products, 2nd set:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prod2_edit">
        <property name="text">
         <string>products2.txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="enterBag1_lbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1st bag:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="bag1_edit">
        <property name="text">
         <string>bag1.txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="enterBag2_lbl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2nd bag:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="bag2_edit">
        <property name="text">
         <string>bag2.txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="enterMProd_lbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>merged products:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mergedProd_edit">
        <property name="text">
         <string>mergedProducts.txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="enterMBags_lbl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>merged bags:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mergedBags_edit">
        <property name="text">
         <string>mergedBags.txt</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="enterNames1_lbl">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>66</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>enter the names</string>
       </property>
      </widget>
      <widget class="QPushButton" name="findObject_btn">
       <property name="geometry">
        <rect>
         <x>53</x>
         <y>600</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>find item</string>
       </property>
      </widget>
      <widget class="QLabel" name="enterNames2_lbl">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>98</y>
         <width>120</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>of your files:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openFiles_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>open files</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="findObject_edit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>560</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
